### ISSUE TITLE
Allow to import passphrase protected  BIP-38 PrivKeys via NFC

### ISSUE THREAD

**Issue by austrocoin** [mycelium-com/wallet#196]
I was also looking too for a way to import BIP-38 PrivKeys into Mycelium. Now it`s posible in-app via QR scan, clipboard and trezor But is it possible via an external app? Found a reddit and it seems it worked in that example. It must be done via an extra app in this case for scanning NFC and call Mycelium with just a simple data-command in format "bitcoin:Privkey_bip32" explained here:

https://www.reddit.com/r/Bitcoin/comments/2n0k3i/breaking_bitcoin_nfc_payments_now_secure_bip38/

The passphrase should then be asked in Mycelium before finally Key is imported. It would be easy then to import encrypted PrivKeys via an app or NFC. Any idea how I call Mycelium to import an encrypted Private Key with this format "bitcoin:PrivKey_bip32" from an external app? 

Thanks

**Comment by jandreske** [comment]
The bip38 via nfc capability is not on the regular release, that one wasn't a custom prototype and due to lack of development resources and a full list of stuff to do it never made it to the main build. 
But scanning QR codes with bip38 encrypted private keys should work just fine - what problems are you experiencing?

-------- Ursprüngliche Nachricht --------
Von: austrocoin <notifications@github.com> 
Datum: 01.10.2015  14:29  (GMT+01:00) 
An: mycelium-com/wallet <wallet@noreply.github.com> 
Betreff: [wallet] [Question] - Is there a way to import "BIP-38 PrivKeys" from
  an external app into Mycelium? (#196) 

I was also looking too for a way to import BIP-38 PrivKeys into Mycelium. Nor it`s posible via QR scan, clipboard and trezor But is it possible via an app? Found a reddit and it seems it worked in that example. It must be done via an extra app in this case for scanning NFC and call Mycelium with just a simple data-command in format "bitcoin:Privkey_bip32" explained here:

https://www.reddit.com/r/Bitcoin/comments/2n0k3i/breaking_bitcoin_nfc_payments_now_secure_bip38/

The passphrase should then be asked in Mycelium before finally Key is imported. It would be easy then to import encrypted PrivKeys via an app or NFC. Any idea how I call Mycelium to import an encrypted Private Key with this format "bitcoin:PrivKey_bip32" from an external app? 

Thanks

**Comment by austrocoin** [comment]
Yes - you describe exactly I was looking for. The import via NFC I could not find in-app, I tested the QR import what works fine. My question is if there would be a way to solve it via an external app... what scan it and give a command to Mycelium for importing? Thanks for the quick and informative answer.

**Comment by swirlsnap** [comment]
I would love this feature as well and if possible could you provide the apk of the release that did have this functionality? @jandreske

**Comment by swirlsnap** [comment]
The main use case for this for me is storing an encrypted wallet address on an implanted NFC chip -- without the ability to scan to import on the cold storage spending menu, you first need to use an NFC tool and then copy the address, and then paste it in -- not very ideal =/

**Comment by Giszmo** [comment]
duplicate of #156
